apis/python/src/tiledbsoma/io/ingest.py

```diff
@@ -886,7 +886,7 @@ def append_X(
     with ``update_obs`` and ``update_var``, as an itemized alternative to doing
     ``from_anndata`` with a registration mapping supplied.
 
-    Example:
+    Example::
```

Comment:
Sphinx grossly misrenders this unless we have the double colon. Slightly uglier to have a double colon for help(foo) in Python; much uglier to not have the double colon for Sphix.